frontend/src/pages/HomePage.js

```diff
@@ -35,8 +35,4 @@ const Titel = styled.section`
   justify-content: center;
   margin-top: 20px;
   height: 10%;
```

Comment:
Maybe better use padding to give the element more room to _breath_. Not sure what you are aiming for.. Letting the layout component (`Page`) determine children size is usually more robust and reusable.